openpype/hosts/blender/utility_scripts/make_paths_relative.py

```diff
@@ -11,19 +11,18 @@
         f"Blend file | All paths converted to relative: {bpy.data.filepath}"
     )
     # Resolve path from source filepath with the relative filepath
-    datablocks_with_filepath = [
-        datablock
-        for datablock in list(bpy.data.libraries) + list(bpy.data.images)
-        if not datablock.is_library_indirect
-    ]
-    for datablock in datablocks_with_filepath:
+    for datablock in list(bpy.data.libraries) + list(bpy.data.images):
```

Comment:
Here you may want to use `chain` instead of casting two lists, which is heavy:
```suggestion
    for datablock in itertools.chain(bpy.data.libraries, bpy.data.images):
```